java-package/mxnet-engine/src/main/include/mxnet/c_api_1_8.h

```diff
@@ -0,0 +1,3422 @@
+/*
```

Comment:
why keeping this?

Reply:
All fixed.
- Remove unnecessary prefix _Mx_ for a few Classes;
- Update copyright for this module;
- Take the _c_api.h_ directly from the upper directory.